backend/tournesol/models/entity.py

```diff
@@ -163,6 +163,32 @@ def link_to_youtube(self):
             '<a href="https://youtu.be/{}" target="_blank">Play ▶</a>', self.video_id
         )
 
+    def criteria_scores_distributions(self, poll):
+        """Returns the distribution of critera score for the entities"""
+
+        # Fetch data with QuerySet
+        contributor_rating_criteria_score_list = [
+            [y for y in x.criteria_scores.all()]
+            for x in self.contributorvideoratings.filter(poll=poll)]
+
+        contributor_rating_criteria_score_flatten_list = [
+            val for _list in contributor_rating_criteria_score_list for val in _list]
+
+        # Format data into dictionnary
+        scores_dict = dict()
+        for element in contributor_rating_criteria_score_flatten_list:
+            if element.criteria not in scores_dict:
+                scores_dict[element.criteria] = []
+            scores_dict[element.criteria].append(element.score)
+
+        # Create object
+        criteria_distributions = []
+        for key, value in scores_dict.items():
+            distribution, bins = np.histogram(np.array(value))
```

Comment:
By default, `np.histogram` will return 10 bins between the min and max values in the array. Don't we want to use a fixed `range` here, to make the comparison between different entities easier?

cc @lfaucon what do you think?

Reply:
Probably yes. For the radarchart and the barchart we fixed the range to [-1, 1] (even if some ratings can be >1 or <-1). It would be best to do the same here.

Reply:
Ok I added a default bins with a fixed range of `[-1,1]` if the value are inside this range. If not the default np historgram behaviour is selected.

Does it look for you or do you want a different behaviour ? @amatissart @lfaucon

Reply:
I think it would really makes things harder to handle if the ranges can vary. I made a suggestion below to propose that the number of values out of bounds are instead counted in the two extreme buckets of the histogram. That corresponds to how we will want to display it on the frontend or extension